.gitignore

```diff
@@ -140,3 +140,5 @@ gradle-app.setting
 .idea/jarRepositories.xml
 .idea/compiler.xml
 .idea/codeStyles/
+.idea/checkstyleidea-libs/
+.idea/checkstyle-idea.xml
```

Comment:
Just curious, is there a reason not to ignore .idea entirely?

Reply:
The reason was that it is somewhat interesting to get noticed when the IDE creates files that developers of gitignore.io have considered important enough to be included in repo. But now that this list keeps growing, I think it is good idea just ignore the whole dir.

Reply:
Apparently it [could be useful](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) to share some files, but since we currently ignore it completely anyways, I'm for ignoring the whole dir.